nbs/examples/Robust_Regression.ipynb

```diff
@@ -0,0 +1,786 @@
+{
```

Comment:
<p>Add the term  "to outliers" in the title somewhere, there are many things a model can be robust to.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Nixtla/neuralforecast/pull/582/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>